app/controllers/organizations_controller.rb

```diff
@@ -4,4 +4,11 @@ class OrganizationsController < ApplicationController
   def index
     @organizations = Organization.all
   end
+
+  def conferences
+    @organization = Organization.find(params[:organization_id])
```

Comment:
This will be loaded by cancancan on https://github.com/bear454/osem/blob/636437f277689ea1ff2d1fea87288a7829315e14/app/controllers/organizations_controller.rb#L2 if the params have id instead of organization_id, which depends on the routes.